src/vendor/modules.txt

```diff
@@ -71,6 +71,10 @@ github.com/Microsoft/go-winio/internal/fs
 github.com/Microsoft/go-winio/internal/socket
 github.com/Microsoft/go-winio/internal/stringbuffer
 github.com/Microsoft/go-winio/pkg/guid
+# github.com/confluentinc/confluent-kafka-go v1.9.2
```

Comment:
We've been vendorizing packages (https://github.com/open-lambda/open-lambda/tree/main/src/vendor) so that we have everything necessary to build this project in our own repo.  Copies of everything makes our repo bigger, but less vulnerable to dependencies going away.

Reply:
If vendoring adds a bunch of files (probably will), let's see if we can do that in its own PR.